syntheseus/cli/search.py

```diff
@@ -79,6 +80,27 @@ class MCTSConfig:
     bound_function_class: str = "pucb_bound"
 
 
+@dataclass
+class PDVNConfig:
+    max_expansion_depth: int = 10
+
+    value_function_syn_class: str = "ConstantNodeEvaluator"
+    value_function_syn_kwargs: Dict[str, Any] = field(default_factory=lambda: {"constant": 0.5})
+
+    value_function_cost_class: str = "ConstantNodeEvaluator"
+    value_function_cost_kwargs: Dict[str, Any] = field(default_factory=lambda: {"constant": 0.0})
+
+    and_node_cost_fn_class: str = "ConstantNodeEvaluator"
+    and_node_cost_fn_kwargs: Dict[str, Any] = field(default_factory=lambda: {"constant": 0.1})
+
+    policy_class: str = "ReactionModelProbPolicy"
+    policy_kwargs: Dict[str, Any] = field(default_factory=dict)
```

Comment:
Is it then the case that it's not yet possible to plug in a PDVN-improved policy here (and so for now PDVN would be expected to behave similarly to MCTS)?

Reply:
OK, so essentially the version we integrate here does not allow one to reproduce the results from the original PDVN paper, as that would require plugging in _two_ models (i.e. a separate policy model different from the reaction model), which the current integration does not allow, right? But this is fine as a stepping-stone towards full integration.

Reply:
Yeah, it's true